src/containers/PrimeStatusBanner/__tests__/index.spec.tsx

```diff
@@ -17,6 +17,9 @@ describe('PrimeStatusBanner', () => {
         isPrime: true,
       },
     }));
+    (useGetPrimeClaimWaitingPeriod as Vi.Mock).mockImplementation(() => ({
+      data: 600,
+    }));
 
```

Comment:
I'd recommend adding some tests to cover the different scenarios, now that we're fetching everything from contracts